pkg/clusters/addons/kuma/addon.go

```diff
@@ -226,7 +226,7 @@ spec:
 // enableMTLS attempts to apply a Mesh resource with a basic retry mechanism to deal with delays in the Kuma webhook
 // startup
 func (a *Addon) enableMTLS(ctx context.Context, cluster clusters.Cluster) (err error) {
-	for i := 0; i < 5; i++ {
+	for i := 0; i < 12; i++ {
```

Comment:
It just occurred to me that this code block might be better off when implemented with a `select` on `ctx.Done()` and `time.After()` channels so that it can be interrupted, contrary to `time.Sleep()`.

Reply:
Yes, we may need change this part to use `select` and `time.After`.